app/driver/settings.py

```diff
@@ -351,3 +351,6 @@
         'rest_framework.authentication.SessionAuthentication',
         'rest_framework.authentication.TokenAuthentication',
     )
+
+# These fields will be visible to read-only users
+READ_ONLY_FIELDS = ['Accident Details']
```

Comment:
Yeah, that was something I was thinking about -- it's possible that `Accident` might be renamed, but the client may also want to do something like making anonymized information on Passengers or Vehicles available to the public, in which case we'd need to find another approach. I'm not sure yet what is most likely to happen, so I went with this as something that would allow us to address a fairly wide variety of changes (relatively) quickly, and we can come up with tailor-made solutions once we know more about the likely use cases.